spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -135,7 +135,7 @@ public static Object[][] parameters() {
         true,
         WRITE_DISTRIBUTION_MODE_NONE,
         false,
-        null,
+        "test",
```

Comment:
I making on additional parameterized test group test branches but also write via Parquet because previously we only tested branches with Avro writes and that wouldn't exericse the metadata delete tests which would've caught this.

Reply:
Why does Avro not exercise this case?

Reply:
Right, we don't have the upper/lower bounds on columns so evaluation of whether the file can be deleted just based off the delete predicate fails. So as far as I can tell metadata deletes aren't possible with Avro data files.